src/core/include/openvino/op/i420_to_rgb.hpp

```diff
@@ -34,6 +34,8 @@ class OPENVINO_API I420toRGB : public util::ConvertColorI420Base {
 public:
     OPENVINO_OP("I420toRGB", "opset8", util::ConvertColorI420Base);
 
+    BWDCMP_RTTI_DECLARATION;
```

Comment:
@ilyachur

Reply:
why do we need it? we usually should not use old RTTI

Reply:
@ilya-lavrenov Otherwise we have a segfault on ngraph deinitialization. This fix WA suggested by @ilyachur un unblock our work :)

Reply:
@Lyamin-Roman @vladimir-paramuzov do we have a ticket for this issue?